Modules/FindCUDA.cmake

```diff
@@ -2,11 +2,10 @@
 # FindCUDA
 # --------
 #
-# Tools for building CUDA C files: libraries and build dependencies.
-#
 # This script locates the NVIDIA CUDA C tools.  It should work on linux,
 # windows, and mac and should be reasonably up to date with CUDA C
 # releases.
+# The approach follows that taken in Caffe: caffe.berkeleyvision.org/
```

Comment:
What approach?  And by the way, I'm not sure what you are attributing to Caffe, but perhaps you could be more specific.  Also, it would be nice to have a line break before a new paragraph, since I use autoformatting on comments, and it would merge this with the previous paragraph.  Add an extra '#' line in front of new paragraphs.